src/plugins/intel_gpu/src/plugin/ops/interpolate.cpp

```diff
@@ -49,7 +49,10 @@ static void CreateInterpolateOp(Program& p, const std::shared_ptr<ngraph::op::v4
         }
     }
 
-    OPENVINO_ASSERT(!scales_constant || axes.size() == scales.size(), op->get_friendly_name(), " Incorrect axes and scales should be the same size");
+    bool correct_params = !scales_constant;
+    correct_params |= axes.size() == scales.size();
+    correct_params |= attrs.shape_calculation_mode == ov::op::v4::Interpolate::ShapeCalcMode::SIZES;
+    OPENVINO_ASSERT(correct_params, "[GPU] Incorrect axes and scales values for Interpolate operation with id ", op->get_friendly_name());
```

Comment:
Is there any reason why only "scales_constant" input is validated, but there were no such checks for the "sizes_constant" ?